openapi/ga4gh-tool-discovery.yaml

```diff
@@ -391,6 +394,13 @@ paths:
           description: >-
             An identifier of the tool version for this particular tool registry,
             for example `v1`.
+        - name: format
+          in: query
+          required: false
+          type: string
+          description: specify format=zip to download a zip file of all files available
```

Comment:
1. All other descriptions start with a capital. And usually end with a period.
1. Descriptions are also generally not imperative, although that does exist in some cases.
1. "all files available" implies there are files that aren't available.

Perhaps, something like, although this isn't right either:

```
If zip, downloads a zip file with all of the version's files for the specified type.
```

Same comment for OpenAPI file

Reply:
(FYI, the OpenAPI is auto-generated)

Reply:
The endpoint already has 3 required path parameters: {id}, {version_id}, and {type}.  This endpoint should generally imply it's specific to those 3 properties. Having a query parameter's description single out 2/3 properties is unusual and redundant.

Reply:
"If zip"  implies format could be something other than zip